torch/lib/THNN/generic/VolumetricAdaptiveAveragePooling.c

```diff
@@ -0,0 +1,302 @@
+#ifndef TH_GENERIC_FILE
+#define TH_GENERIC_FILE "generic/VolumetricAdaptiveAveragePooling.c"
+#else
+
+#define START_IND(a,b,c) (int)floor((float)(a * c) / b)
+#define END_IND(a,b,c) (int)ceil((float)((a + 1) * c) / b)
+// #define START_IND(a,b,c) a * c / b
+// #define END_IND(a,b,c)  (a + 1) * c / b + ((a + 1) * c % b > 0)?1:0
+
+static void THNN_(VolumetricAdaptiveAveragePooling_updateOutput_frame)(
+          real *input_p,
+          real *output_p,
+          long nslices,
+          long isizeT,
```

Comment:
can you just use the same variable names / name formats as in SpatialAdaptiveAveragePooling?  Even if your names are better (they are), this creates unnecessary noise in review and maintenance.

Reply:
I debated against myself on that for some time too. But I wanted to be consistent between the cpu and the cuda var names. I really didn't want to follow the spatial [cuda var names](https://github.com/pytorch/pytorch/blob/master/torch/lib/THCUNN/SpatialAdaptiveAveragePooling.cu#L21) which leads to `input_t` being a var rather than a type. What's your though on this?

Reply:
if that's the case, just codemod the names in Spatial then to isize*.

Reply:
Should I do that in this PR or a separate one?

Reply:
doing in this PR is fine.

Reply:
Will do tomorrow.